src/utm_codes.js

```diff
@@ -12,14 +12,15 @@ window.addEventListener('load', function() {
     utmParams = ['utm_source', 'utm_medium', 'utm_content', 'utm_campaign'];
 
     $.each(utmParams, function(_paramsIndex, utmParam) {
-      var utmParamValue;
+      var utmParamValue, utmFieldID;
```

Comment:
If you want to do this less specific to our use case, you could try something like this
```
$.each($forms, function(_formIndex, form) {
  ...
  paramName = $(form).data('utm-tracker-param-name');
  ...
   if(paramName)
     utmFieldID = paramName + '[' + utmParam + ']';
   else
     utmField = utmParam
```

So you can define the form something like this
`<form class='utm-tracker-requried' data-utm-tracker-param-name='contact'>`